py/picca/bal_tools.py

```diff
@@ -0,0 +1,67 @@
+##Functions used for masking BAL features ##LE 2020
+
+
+import fitsio
+import scipy as sp
+
+def read_bal(bal_catalog): ##Based on read_dla from picca/py/picca/io.py
+    lst = ['THING_ID','VMIN_CIV_450','VMAX_CIV_450','VMIN_CIV_2000','VMAX_CIV_2000']
+    h = fitsio.FITS(bal_catalog)
+    bal_dict = { k :h[1][k][:] for k in lst }
+    h.close()
+
+    return bal_dict
+
+
+#based on add_dla from picca/py/picca/data.py
+def add_bal_rf(bal_catalog,thingid,bal_index): #BAL catalog, THING_ID, and BAL index
+    ### Wavelengths in Angstroms
+    lines = {
```

Comment:
Consider moving `lines` to `picca.constants` or adding them as a module variable, and pass them to the function as a keyword variable with a default. This is to provide flexibility in case we need to test the usage of different lines.

Reply:
Hi @LaurEnnesser,
Have you considered this?